hw/top_englishbreakfast/util/sw_sources.patch

```diff
@@ -5,7 +5,7 @@ index 99485c5ff..1c8e29d86 100644
 @@ -81,19 +81,6 @@ _reset_start:
  _start:
    .globl _start
-
+ 
```

Comment:
There seem to be a bunch of extra spaces being inserted. I guess you are generating the patch in a slightly different way maybe? Might be worth trying to get rid of them to make the diff smaller.

Reply:
Spaces removed.